frontend/src/components/form/api/ApiSelect.vue

```diff
@@ -24,7 +24,7 @@ Displays a field as a e-select + write access via API wrapper
     >
       <template #append>
         <v-icon>mdi-menu-down</v-icon>
-        <api-wrapper-append :wrapper="wrapper" />
+        <ApiWrapperSelectAppend :wrapper="wrapper" :resettable="resettable" />
```

Comment:
When I click "Retry" in the Select, the drowdown opens. Do we need to have some @click.stop somewhere?

Reply:
I also noticed this in #3144 (the very last point)

Reply:
This happened with the checkbox as well because of the label. I'll have a look.